### PR TITLE
fix widget compile error

### PR DESCRIPTION
**Pull request by SiyaoHuang**

### CODE REVIEW

File: com.unity.uiwidgets/Runtime/widgets/basic.cs [hunk 1]
```diff
@@ -1901,8 +1901,8 @@ public class RawImage : LeafRenderObjectWidget {
                 alignment: alignment,
                 repeat: repeat,
                 centerSlice: centerSlice,
-                invertColors: invertColors,
-                filterMode: filterMode
+                invertColors: invertColors
+                // filterMode: filterMode
```
Comment (zhuxingwei) [inline com.unity.uiwidgets/Runtime/widgets/basic.cs:1905]: we should add the filterQuality parameter here， making it exactly the same with the codes in flutter?

File: com.unity.uiwidgets/Runtime/widgets/texture.cs [hunk 2]
```diff
@@ -12,20 +12,20 @@ public class Texture : LeafRenderObjectWidget {
         //         instance.scheduleFrame(false);
         //     }
         // }
-        
-        public Texture(Key key = null, UnityEngine.Texture texture = null) : base(key: key) {
-            D.assert(texture != null);
-            this.texture = texture;
+
+        public Texture(int textureId, Key key = null) : base(key: key) {
```
Comment (zhuxingwei) [inline com.unity.uiwidgets/Runtime/widgets/texture.cs:16]: the sequence of parameters are not correct？ should it be (key , textureId)

File: com.unity.uiwidgets/Runtime/widgets/texture.cs [hunk 3]
```diff
@@ -12,20 +12,22 @@ public class Texture : LeafRenderObjectWidget {
...
         //     }
         // }
-        
-        public Texture(Key key = null, UnityEngine.Texture texture = null) : base(key: key) {
-            D.assert(texture != null);
-            this.texture = texture;
+
+        public Texture( Key key = null, int? textureId = null) : base(key: key) {
+            D.assert(textureId != null);
+            if (textureId != null) {
```
Comment (zhuxingwei) [inline com.unity.uiwidgets/Runtime/widgets/texture.cs:18]: this if can be removed?

File: com.unity.uiwidgets/Runtime/widgets/basic.cs (hunk 1 above)
Comment (SiyaoHuang) [reply com.unity.uiwidgets/Runtime/widgets/basic.cs]: Updated

File: com.unity.uiwidgets/Runtime/widgets/texture.cs (hunk 2 above)
Comment (SiyaoHuang) [reply com.unity.uiwidgets/Runtime/widgets/texture.cs]: Updated

File: com.unity.uiwidgets/Runtime/widgets/basic.cs [hunk 4]
```diff
@@ -1860,7 +1860,9 @@ public class RawImage : LeafRenderObjectWidget {
             ImageRepeat repeat = ImageRepeat.noRepeat,
             Rect centerSlice = null,
             bool invertColors = false,
-            FilterMode filterMode = FilterMode.Bilinear
+            FilterMode filterMode = FilterMode.Bilinear ,
```
Comment (zhuxingwei) [inline com.unity.uiwidgets/Runtime/widgets/basic.cs:1863]: filterMode can be removed?

**Review by zhuxingwei — APPROVED** [review]

**Pull request merged**